src/Contact_agent/Contact.java

```diff
@@ -9,4 +9,33 @@ public Contact(String name, String telephone) {
         this.name = name;
         this.name = telephone;
     }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getTelephone() {
+        return telephone;
+    }
+
+    public void setTelephone(String telephone) {
+        this.telephone = telephone;
+    }
+
+    public boolean equals(Contact cont) {
```

Comment:
add @Override
Also, maybe you haven't learnt it yet, but if you override equals, you should ALWAYS override hashCode. But don't worry about it now if you haven't learnt it yet.